crates/trie/src/hash_builder/mod.rs

```diff
@@ -8,14 +8,13 @@ use reth_primitives::{
     trie::{BranchNodeCompact, TrieMask},
     H256,
 };
-use std::fmt::Debug;
-use tokio::sync::mpsc;
+use std::{fmt::Debug, sync::mpsc};
 
 mod value;
 use value::HashBuilderValue;
 
 /// A type alias for a sender of branch nodes.
-pub type BranchNodeSender = mpsc::UnboundedSender<(Nibbles, BranchNodeCompact)>;
+pub type BranchNodeSender = mpsc::Sender<(Nibbles, BranchNodeCompact)>;
```

Comment:
are there docs on this, I see the hashbuilder takes a sender for nibbles, but unclear what this is actually used for?

Reply:
amended docs